source/extensions/common/wasm/v8/v8.cc

```diff
@@ -56,22 +56,29 @@ class V8 : public WasmVm {
   std::unique_ptr<WasmVm> clone() override { return nullptr; }
 
   void start(Context* context) override;
-  absl::string_view getMemory(uint32_t pointer, uint32_t size) override;
-  bool getMemoryOffset(void* host_pointer, uint32_t* vm_pointer) override;
-  bool setMemory(uint32_t pointer, uint32_t size, void* data) override;
+  absl::string_view getMemory(uint64_t pointer, uint64_t size) override;
+  bool getMemoryOffset(void* host_pointer, uint64_t* vm_pointer) override;
+  bool setMemory(uint64_t pointer, uint64_t size, void* data) override;
+  bool setWord(uint64_t pointer, uint64_t word) override;
 
 #define _REGISTER_HOST_GLOBAL(_type)                                                               \
   std::unique_ptr<Global<_type>> makeGlobal(absl::string_view moduleName, absl::string_view name,  \
                                             _type initialValue) override {                         \
     return registerHostGlobalImpl(moduleName, name, initialValue);                                 \
   };
-  _REGISTER_HOST_GLOBAL(uint32_t);
   _REGISTER_HOST_GLOBAL(double);
+  _REGISTER_HOST_GLOBAL(Word);
+#if 0
```

Comment:
Could you remove the `#if 0`?

Also, extra nitpick, could you move `_REGISTER_HOST_GLOBAL(Word);` before `_REGISTER_HOST_GLOBAL(double);` to match order in other places? Thanks.

Reply:
done